Xabe.FFmpeg/Probe/IProbe.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading.Tasks;
+
+namespace Xabe.FFmpeg
+{
+    /// <summary>
+    ///     Allows to prepare and start IProbe.
+    /// </summary>
+    public interface IProbe
+    {
+        /// <summary>
+        /// Start probe with result from console
+        /// </summary>
+        /// <param name="args">Args to pass to FFprobe</param>
+        /// <returns>Console output output</returns>
```

Comment:
"Output from console in specified format"

Reply:
done